src/testdir/test_options.vim

```diff
@@ -1465,5 +1467,41 @@ func Test_endoffile_default()
   call delete('Xtestout')
 endfunc
 
+" Test for setting the 'lines' and 'columns' options to a minimum value
+func Test_set_min_lines_columns()
+  let lines =<< trim END
+    let save_lines = &lines
+    let save_cols = &columns
+    set nomore
+    let msg = []
+    let v:errmsg = ''
+    silent! let &columns=0
+    call add(msg, v:errmsg)
+    silent! set columns=0
+    call add(msg, v:errmsg)
+    silent! call setbufvar('', '&columns', 0)
+    call add(msg, v:errmsg)
+    call writefile(msg, 'XResultsetminlines')
+    silent! let &lines=0
+    call add(msg, v:errmsg)
+    silent! set lines=0
+    call add(msg, v:errmsg)
+    silent! call setbufvar('', '&lines', 0)
+    call add(msg, v:errmsg)
+    call writefile(msg, 'XResultsetminlines')
+    let &lines = save_lines
+    let &columns = save_columns
+  END
+  call writefile(lines, 'XTestMinLines', 'D')
+  let cmd = $"{GetVimCommand()} --clean -S XTestMinLines -c qa!"
+  call system(cmd)
```

Comment:
It seems that `RunVim()` can be used here (it is also used in the test above):
```suggestion
    qa!
  END
  if !RunVim([], lines, '--clean')
    return
  endif
```

Reply:
Yes. Thanks for the suggestion. I have updated the PR with this change.